src/request_handler.rs

```diff
@@ -59,6 +69,16 @@ impl RequestHandler {
 
     Ok(Response::new(Body::from(body.into_string())))
   }
+
+  async fn serve_file(&self, file_path: &str) -> Result<Response<Body>> {
+    // todo: stream files
```

Comment:
Currently we're loading whole files into memory. Which is probably bad.

Reply:
Agree that it's bad. I think we can open an issue and leave it for another PR. The person running the binary controls the contents of the www dir, so they can just not put large files in there for now :P

When we do want to do streaming, it looks like we'll want `Body::channel`, which returns a channel into which bytes can be written, or `Body::wrap_stream`, if we have a `Stream` implementation that will return chunks of the file.